GTG/backends/__init__.py

```diff
@@ -119,7 +119,7 @@ def get_new_backend_dict(self, backend_name, additional_parameters={}):
         """
         if backend_name not in self.backend_modules:
             return None
-        dic = {}
+        dic = {'first_run': True}
```

Comment:
Without it, loading the `caldav` backend gives that : 
```
Traceback (most recent call last):
  File "/home/jaes/devel/gtg/.local_build/install/lib/python3.9/site-packages/GTG/gtk/backends/addpanel.py", line 152, in on_confirm
    self.dialog.on_backend_added(backend_name)
  File "/home/jaes/devel/gtg/.local_build/install/lib/python3.9/site-packages/GTG/gtk/backends/__init__.py", line 245, in on_backend_added
    self.req.register_backend(backend_dic)
  File "/home/jaes/devel/gtg/.local_build/install/lib/python3.9/site-packages/GTG/core/requester.py", line 251, in register_backend
    return self.ds.register_backend(dic)
  File "/home/jaes/devel/gtg/.local_build/install/lib/python3.9/site-packages/GTG/core/datastore.py", line 430, in register_backend
    first_run = backend_dic["first_run"]
KeyError: 'first_run'
```
It has been made mandatory but not in this branch. Solution to avoid that is either to `backend_dic.get` or to preload the `backend_dic` with valid values. I chose the later. 
We might also take care of that in another branch before merging this one.

Reply:
Ah right, we can take care of that later